site/src/pages/DeploySettingsPage/OAuth2AppsSettingsPage/OAuth2AppsSettingsPageView.tsx

```diff
@@ -28,12 +29,14 @@ type OAuth2AppsSettingsProps = {
   apps?: TypesGen.OAuth2ProviderApp[];
   isEntitled: boolean;
   isLoading: boolean;
+  error?: unknown;
```

Comment:
Should this be optional? Feels like making a dev always pass a value in (even if it's hopefully `undefined` 99% of the time) would help avoid forgetting something down the line

Reply:
Good point, I just assumed it could be undefined from React Query but I did not actually check.  :stuck_out_tongue:

Reply:
Ah I see, it can be `null` but not `undefined`.  Fixing.